### DISCUSSION TITLE
Multiple selection in dragmode absolute

### DISCUSSION THREAD

**Discussion by badiuciprian** [GrapesJS/grapesjs#5264] [Q&A]
Hi guys, I saw that multiple selection is available but not working in dragmode absolute. I've searched the Discussions and Issues but could not find any related topics. Is there a bug or it requires something special to work?
T'hanks

**Comment by artf** [comment]
Yeah, known bug, there are actually a lot of issues with the absolute dragmode so we track them [here](https://github.com/GrapesJS/grapesjs/issues/3770)

**Comment by badiuciprian** [comment]
ok, thank you!
I'll keep an eye on the updates :)
